frontend-service/src/app/css/App.css

```diff
@@ -1,39 +1,54 @@
-.App {
-  text-align: center;
+@import url("https://fonts.googleapis.com/css2?family=Oxygen:wght@700&family=Varela+Round&display=swap");
+
+body {
+  /* font-family: 'Oxygen', sans-serif; */
+  font-family: 'Varela Round', sans-serif;
```

Comment:
Use 4 spaces for your css indentation, it's easier to read that way

Reply:
Alright